editor/animation_track_editor.cpp

```diff
@@ -91,6 +91,10 @@ void AnimationTrackKeyEdit::_fix_node_path(Variant &value) {
 
 void AnimationTrackKeyEdit::_update_obj(const Ref<Animation> &p_anim) {
 	if (setting || animation != p_anim) {
+		AnimationPlayer *player = AnimationPlayerEditor::get_singleton()->get_player();
+		if (player != nullptr) {
+			player->advance(0.0); // Force an update to refresh the view
```

Comment:
```suggestion
			player->advance(0.0); // Force an update to refresh the view.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).